StortSpelProjekt/Engine/src/Renderer/Renderer.h

```diff
@@ -126,6 +128,7 @@ class Renderer
 	void UnInitCameraComponent(component::CameraComponent* component);
 	void UnInitBoundingBoxComponent(component::BoundingBoxComponent* component);
 	void UnInitGUI2DComponent(component::GUI2DComponent* component);
+	void UnitParticleEmitterComponent(component::ParticleEmitterComponent* component);
```

Comment:
Should this not be called 'UnInit...' instead of 'Unit...'?

Reply:
Good catch, just noticed myself ^^.